### PR TITLE
Improve DevTools>Working with iframes

### PR DESCRIPTION
**Pull request by hamishwillee**
This improves the [Working with iframes](https://developer.mozilla.org/en-us/docs/Tools/Working_with_iframes) and fixes flaws. 

The main benefit is that it states what the feature does in just a little more detail at the top. It also notes that the feature only works on pages with iframes and when setting is enabled.

### CODE REVIEW

File: files/en-us/tools/working_with_iframes/index.html [hunk 1]
```diff
@@ -10,14 +10,26 @@
 ---
 <div>{{ToolsSidebar}}</div>
 
-<p>You can point the developer tools at a specific <a href="/en-US/docs/Web/HTML/Element/iframe"><code>iframe</code></a> within a document.</p>
+<p>You can point the developer tools at a specific <a href="/en-US/docs/Web/HTML/Element/iframe"><code>iframe</code></a> within a document. The <a href="/en-US/docs/Tools/Page_Inspector">Inspector</a>, <a href="/en-US/docs/Tools/Web_Console">Console</a>, <a href="/en-US/docs/Tools/Debugger">Debugger</a> and all other developer tools will then target that iframe (essentially behaving as if the rest of the page does not exist).</p>
 
 <p>{{EmbedYouTube("Me9hjqd74m8")}}</p>
 
-<p>You'll see a button in the toolbar:</p>
+<p>To set an iframe as the target for the developer tools, first select the <em>iframe button</em> to launch a popup listing all the iframes in the document (and the main document itself). Then Select an entry to make it the <em>target iframe</em>. Note that the iframe button is not displayed if the page doesn't have any iframes!</p>
```
Comment (sideshowbarker) [inline files/en-us/tools/working_with_iframes/index.html:17]:
```suggestion
<p>To set an iframe as the target for the developer tools, first select the <em>iframe button</em> to launch a popup listing all the iframes in the document (and the main document itself). Then select an entry to make it the <em>target iframe</em>. Note that the iframe button is not displayed if the page doesn't have any iframes!</p>
```
…assuming that *Select* was not capitalized intentionally; if it is in fact intentional, then please just dismiss this suggestion

File: files/en-us/tools/working_with_iframes/index.html [hunk 2]
```diff
@@ -10,14 +10,26 @@
...
+    <li>Select an entry to make it the <em>target iframe</em>.</li>
+  </ul>
+</p>
 
-<p><img alt="" src="https://mdn.mozillademos.org/files/8549/frame-selection-popup.png" style="display: block; margin-left: auto; margin-right: auto;"></p>
+<p><img alt="Screenshot showing how to set an iframe as the target of developer tools (using the iframe button)" src="developer_tools_select_iframe.png" /></p>
+
+<div class="notecard note">
+  <h4>Note</h4>
+  <p>The iframe context picker button feature is enabled by default (if it has been disabled the iframe button is never displayed). The feature can be re-enabled from the Settings, using checkbox shown below:</p>
```
Comment (sideshowbarker) [inline files/en-us/tools/working_with_iframes/index.html:31]:
```suggestion
  <p>The iframe context picker button feature is enabled by default (if it has been disabled the iframe button is never displayed). The feature can be re-enabled from the Settings menu, using checkbox shown below:</p>
```
…or just dismiss this suggestion if *Settings* without *menu* is actually fine

**Review by sideshowbarker — APPROVED** [review]
Some nittish suggestions but just fine regardless of whether those are applied

**Comment by hamishwillee** [comment]
Thanks for review - changes accepted. Merging.

**Pull request merged**